src/util/psbt/mod.rs

```diff
@@ -91,6 +91,52 @@ impl PartiallySignedTransaction {
     }
 }
 
+#[cfg(feature = "base64")]
+mod display_from_str {
+    use super::PartiallySignedTransaction;
+    use std::fmt::{Display, Formatter, self};
+    use std::str::FromStr;
+    use consensus::encode::{Error, self};
+    use ::base64::display::Base64Display;
+
+    /// Error happening during PSBT decoding from Base64 string
+    #[derive(Debug)]
+    pub enum PsbtParseError {
```

Comment:
If you rebase on `master` this error can also derive `Clone, PartialEq, Eq`.

Reply:
No, since I am limited by `consensus::encode::Error`: https://github.com/rust-bitcoin/rust-bitcoin/blob/master/src/consensus/encode.rs#L48

Reply:
Sorry, that's annoyning, I looked at the wrong error. So I guess this PR is as good as it gets without some breaking changes to make `consensus::encode::Error` implement `Eq`.

Reply:
All possible PRs to implement derives on Error types are already there: #559, #560, #558. Most of that is blocked by upstream. Overall tracking on Error derives across full bitcoin rust repos: https://github.com/orgs/rust-bitcoin/projects/2